sdk/clientcore/core/src/main/java/io/clientcore/core/http/pipeline/HttpRetryPolicy.java

```diff
@@ -125,6 +125,11 @@ public Response<?> process(HttpRequest httpRequest, HttpPipelineNextPolicy next)
         return attempt(httpRequest, next, 0, null);
     }
 
+    @Override
+    public HttpPipelineOrder getOrder() {
```

Comment:
Make these `getOrder()` methods on pillar policies `final`.

Reply:
Same issue with a linting warning, but I'm happy to do it if we want to suppress those warnings.

Reply:
let's make it not final and I can rewrite my enrichment sample in a different way